src/node_url.cc

```diff
@@ -407,29 +407,86 @@ int64_t ParseIPv4Number(const char* start, const char* end) {
   return strtoll(start, nullptr, R);
 }
 
+// https://url.spec.whatwg.org/#ipv4-number-parser
+bool IsIPv4NumberValid(std::string_view input) {
+  // If input is the empty string, then return failure.
```

Comment:
We should remove all comments related to spec, except the spec URL since this is not a common thing in Node.

Reply:
done!